doc/modules/ROOT/pages/rel-embedding-models.adoc

```diff
@@ -53,30 +53,41 @@ Since the prediction parts of the computations in these methods are the same, th
 | Name               | Type                       |
 | source_node_filter | Union[str, int, list[int]] | The specification of source nodes to consider. Either a node label, a node ID, or a list of node IDs
 | target_node_filter | Union[str, int, list[int]] | The specification of source nodes to consider. Either a node label, a node ID, or a list of node IDs
+| relationshipTypes  | str or list[str]           | Relationship types for which the algorithm will be executed
 | top_k              | int                        | How many relationships to produce for each source node. The target nodes with the highest `top_k` scores will be kept for each source node
 | general_config     | **dict[str, Any]           | General GDS algorithm configuration as optional keyword parameters
 |===
 
-In particular, the general algorithm configuration parameters supported as keyword parameters for this algorithm are `concurrency`, `nodeLabels`, `relationshipTypes`, `jobId` and `logProgress`.
+In particular, the general algorithm configuration parameters supported as keyword parameters for this algorithm are `concurrency`, `jobId` and `logProgress`.
```

Comment:
Because `nodeLabels` functionality is covered by `node_source_filter` and `node_target_filer` fields and `relationshipTypes` is covered by `relationship_type` parameter.

Reply:
ok 👍